src/components/branding/iowa-bar/iowa-bar.scss

```diff
@@ -58,12 +63,22 @@ header {
       margin-right: auto;
       margin-left: auto;
     }
+
+    .iowa-bar--narrow.horizontal &,
+    .iowa-bar--narrow.mega & {
+      @include breakpoint(container) {
+        position: relative;
+      }
+    }
```

Comment:
Move to appropriate menu scss file.

Reply:
Maybe this should be moved to the Drupal partial?